editor/src/components/canvas/controls/guideline-controls.tsx

```diff
@@ -153,3 +201,129 @@ function useGuideline<T = HTMLDivElement>(
   }, [innerCallback])
   return controlRef
 }
+
+function framesFromMetadata(
+  metadata: ElementInstanceMetadataMap,
+  paths: Array<ElementPath>,
+): Array<CanvasRectangle> {
+  return mapDropNulls((path) => metadata[EP.toString(path)]?.globalFrame ?? null, paths)
```

Comment:
Please use the functions in `element-metadata-utils.ts` for accessing anything in the metadata

Reply:
thanks, fixed!